update/Pos/Update/Poll/Logic/Apply.hs

```diff
@@ -66,7 +66,7 @@ verifyAndApplyUSPayload ::
     -> m ()
 verifyAndApplyUSPayload lastAdopted verifyAllIsKnown slotOrHeader upp@UpdatePayload {..} = do
     -- First of all, we verify data.
-    either (throwError . PollInvalidUpdatePayload) pure =<< runExceptT (checkUpdatePayload upp)
+    either (throwError . PollInvalidUpdatePayload . show) pure $ runPVerify upp
```

Comment:
1. I don't like that you use `show` in all places where you need to convert `VerError` to `Text`. We want to format error prettily. The right function for it is `pretty`. `show` is not for pretty printing.
2. Why do you `show` here at all? Why not change argument of `PollInvalidUpdatePayload` to `VerError`? It would give more information. It will be more useful when we rewrite error handling in `Pos.Util.Verification` properly.